packages/amplify-graphql-api-construct/jest.config.js

```diff
@@ -2,11 +2,4 @@ const baseConfig = require('../../jest.config.base.js'); // eslint-disable-line
 
 module.exports = {
   ...baseConfig,
-  coverageThreshold: {
```

Comment:
I'm curious why we want to remove the threshold?

Reply:
Because we have a global threshold of 90% in `jest.config.base.js`, from which the package-specific configs extend: https://github.com/aws-amplify/amplify-category-api/blob/main/jest.config.base.js#L16-L21